src/services/item/controller.test.ts

```diff
@@ -0,0 +1,131 @@
+import { StatusCodes } from 'http-status-codes';
+import { v4 as uuid } from 'uuid';
+
+import { FastifyInstance } from 'fastify';
+
+import {
+  AppItemFactory,
+  FolderItemFactory,
+  ItemTagType,
+  PermissionLevel,
+  buildPathFromIds,
+} from '@graasp/sdk';
+
+import build, { clearDatabase } from '../../../test/app';
+import seed from '../../../test/mock';
+import '../../plugins/datasource';
+import { ItemMembership } from '../itemMembership/entities/ItemMembership';
+import { Actor } from '../member/entities/member';
+import { Item } from './entities/Item';
+import { ItemTag } from './plugins/itemTag/ItemTag';
+
+jest.mock('../../plugins/datasource');
+
+describe('Item controller', () => {
+  let app: FastifyInstance;
+  let actor: Actor;
+  beforeEach(async () => {
+    ({ app, actor } = await build());
+  });
+
+  afterEach(async () => {
+    await clearDatabase(app.db);
+    app.close();
+  });
+
+  describe('GET /:id/descendants filters', () => {
+    let uuids;
+    beforeEach(async () => {
+      // uuids[0] is the root
+      // uuids[1] is folder
+      // uuids[2] is hidden
+      // uuids[3] is public
+      uuids = [uuid(), uuid(), uuid(), uuid()];
```

Comment:
What about using distinct constants or a map to increase readability and avoid the comments ?

```suggestion
      const rootUUID = uuid();
      const folderUUID = uuid();
      const hiddenUUID = uuid();
      const publicUUID = uuid();
```